ports/stm32/boards/stm32u5a5xj.ld

```diff
@@ -0,0 +1,35 @@
+/*
+    GNU linker script for STM32U5A5xJ
+*/
+
+/* Specify the memory areas */
+MEMORY
+{
+    FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 4096K
+    FLASH_APP (rx)  : ORIGIN = 0x08008000, LENGTH = 992K    /* sectors 4-127 (for common_bl.ld) */
+    FLASH_FS (rx)   : ORIGIN = 0x08100000, LENGTH = 1024K   /* sectors 128-255 */
+    RAM   (xrw)     : ORIGIN = 0x20000000, LENGTH = 2496K   /* SRAM1/2/3/5: 2496KB */
```

Comment:
Thanks for fixing this, but the fix is in the wrong commit (the second one).  The first commit should have this line corrected.